src/init.ts

```diff
@@ -101,11 +102,26 @@ export class NightwatchInit {
 
     // Generate configuration file
     const configDestPath = await this.getConfigDestPath();
+
     this.generateConfig(answers, configDestPath);
 
+    let mobile_result: MobileResult = {};
+
+    if (answers?.mobile) {
+      if (answers.mobileDevice === 'ios' || answers.mobileDevice === 'both') {
+        const ios = new IosSetup({setup : true, mode: ['real', 'simulator']});
+        mobile_result.ios = await ios.run();
+      }
+
+      if (answers.mobileDevice === 'android' || answers.mobileDevice === 'both') {
+        const android = new AndroidSetup({browsers: answers.mobileBrowsers || ['chrome'], setup: true}, this.rootDir);
```

Comment:
`setup` proper can be omitted from here (we should ask the user after verification if they'd like to setup it or not).